src/Marker.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { Marker as ReactMapGLMarker } from "react-map-gl";
+import { Pin } from './Pin';
+import { categoryColors } from './constants';
+
+const StyledMarker = styled(ReactMapGLMarker)`
+  ${props => props.selected ? 
+    css`
+      z-index: 10;
+      transform: translate(-23px,-40px)
+    ` : css`
+      transform: translate(-15px,-36px);
+    `
+  };
+`;
+
+export const Marker = ({ selected, latitude, longitude, category, onMarkerClick }) => {
+  const pinColor = category
+    ? categoryColors[category].backgroundColor
+    : 'white';
```

Comment:
With this change markers now know about categories... my original intention was to keep that knowledge restricted to `App.js`. It doesn't seem like a big deal to make it globally accessible. Although it does feel out of the scope of this bug fix.

Reply:
I could hoist this back up to App.js pretty easily, I think! And would remove this dependency on the component, which is also good.